File: problems/Excel Sheet Column Title/convertToTitle.cs
public class Solution {
    public string ConvertToTitle(int columnNumber) {
        var stack = new Stack<char>();

        while (0 < columnNumber) {
            stack.Push((char)('A' + --columnNumber % 26));
            columnNumber /= 26;
        }

        return string.Join("", stack);
    }
}

File: problems/Excel Sheet Column Title/convertToTitle.java
class Solution {
    public String convertToTitle(int columnNumber) {
        Stack<Character> stack = new Stack<>();
        StringBuilder sb = new StringBuilder();

        while (0 < columnNumber) {
            stack.push((char)('A' + --columnNumber % 26));
            columnNumber /= 26;
        }

        while (!stack.isEmpty()) {
            sb.append(stack.pop());
        }

        return sb.toString();
    }
}
